src/components/new-safe/steps/Step2/index.tsx

```diff
@@ -78,36 +80,17 @@ const CreateSafeStep2 = ({ onSubmit, onBack, data }: StepRenderProps<NewSafeForm
             </Button>
           </Grid>
           <Grid item xs={12}>
-            <Typography variant="subtitle1" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
-              Safe Mobile owner key (optional){' '}
-              <Tooltip title="TODO: Add tooltip" arrow placement="top">
-                <span style={{ display: 'flex' }}>
-                  <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
-                </span>
-              </Tooltip>
-            </Typography>
-            <Typography variant="body2">
-              Add an extra layer of security and sign transactions with the Safe Mobile app.
-            </Typography>
-          </Grid>
-
-          <Grid item xs={12}>
-            {mobileOwnerFields.map((field, i) => (
-              <OwnerRow
-                key={field.id}
-                groupName={CreateSafeStep2Fields.mobileOwners}
-                index={i}
-                remove={removeMobileOwner}
-              />
-            ))}
-            <Button
-              variant="text"
-              onClick={() => appendMobileOwner({ name: '', address: '' }, { shouldFocus: true })}
-              startIcon={<SvgIcon component={AddIcon} inheritViewBox fontSize="small" />}
-              size="large"
-            >
-              Add mobile owner
-            </Button>
+            <Box p={2} sx={{ backgroundColor: 'background.main', borderRadius: '8px' }}>
+              <Typography variant="subtitle1" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
+                Safe Mobile owner key (optional){' '}
+                <Tooltip title="TODO: Add tooltip" arrow placement="top">
```

Comment:
Do we have a tooltip `title` yet?

Reply:
No. I think it can also figure out in a different PR. wdyt?

Reply:
I'm waiting for the urls to include in the tooltip link

Reply:
I would suggest we include the links in the standard text and include a summary in the tooltips. It's odd to "hide" links. cc @liliiaorlenko

Reply:
Can we go with a short tooltip copy here instead of links? Also let's capitalize on the tips that we are displaying on the side.